src/helix-ui/elements/HXDisclosureElement.js

```diff
@@ -17,15 +17,12 @@ export class HXDisclosureElement extends HXElement {
         return super.observedAttributes.concat([ 'aria-expanded' ]);
```

Comment:
This should be `static get $observedAttributes` and should not call `super.observedAttributes`.